lib/model/content.dart

```diff
@@ -792,6 +894,16 @@ class _ZulipContentParser {
     }
 
     if (localName == 'p' && classes.isEmpty) {
+      // Oddly, the way a math block gets encoded in Zulip HTML is inside a <p>.
+      if (element.nodes case [dom.Element(localName: 'span') && var child]) {
```

Comment:
Hmm, I'm seeing a block-quoted math block with more than one child inside the `<p>`. The HTML has the expected span and then `<br>\n`.

https://chat.zulip.org/#narrow/stream/7-test-here/topic/Chris/near/1714716

`````
````quote
```math
1 + 1
```
````
`````

```
"<blockquote>\n<p><span class=\"katex-display\"><span class=\"katex\"><span class=\"katex-mathml\"><math xmlns=\"http://www.w3.org/1998/Math/MathML\" display=\"block\"><semantics><mrow><mn>1</mn><mo>+</mo><mn>1</mn></mrow><annotation encoding=\"application/x-tex\">1 + 1</annotation></semantics></math></span><span class=\"katex-html\" aria-hidden=\"true\"><span class=\"base\"><span class=\"strut\" style=\"height:0.7278em;vertical-align:-0.0833em;\"></span><span class=\"mord\">1</span><span class=\"mspace\" style=\"margin-right:0.2222em;\"></span><span class=\"mbin\">+</span><span class=\"mspace\" style=\"margin-right:0.2222em;\"></span></span><span class=\"base\"><span class=\"strut\" style=\"height:0.6444em;\"></span><span class=\"mord\">1</span></span></span></span></span><br>\n</p>\n</blockquote>"
```

Reply:
Good find, thanks! I'll handle that, and also take this as a sign I should do some broader experimenting.